models/domains_tlds.php

```diff
@@ -1106,13 +1106,12 @@ private function assignFeatures($package_id, array $vars)
                         'message' => Language::_('DomainsTlds.!error.feature.unsupported', true, $feature)
                     ]
                 ]);
-                continue;
             } else {
                 // Set the package meta
                 $fields = [
                     'package_id' => $package_id,
                     'key' => $feature,
-                    'value' => $vars[$feature],
```

Comment:
This will remove the epp_code feature if it isn't submitted.  I don't think this is what we want. Instead, update
```
 } else {
```
To
```
            } elseif (isset($vars[$feature])) {
```